unit-tests/func/func-common.h

```diff
@@ -169,3 +211,28 @@ inline void do_while_streaming( rs2::sensor depth_sens,
     depth_sens.stop();
     depth_sens.close();
 }
+
+rs2::stream_profile get_profile_by_stream_parameters( rs2::sensor s,
+                                                      rs2_stream stream,
+                                                      rs2_format format,
+                                                      int width,
+                                                      int height,
+                                                      int fps,
+                                                      int stream_index )
```

Comment:
I was hoping you'd put in a default for at least `stream_index`.
```
                 if ((tag.stream == RS2_STREAM_ANY || vp->get_stream_type() == tag.stream) &&
                     (tag.format == RS2_FORMAT_ANY || vp->get_format() == tag.format) &&
                     (tag.width == -1 || vp->get_width() == tag.width) &&
                     (tag.height == -1 || vp->get_height() == tag.height) &&
                     (tag.fps == -1 || vp->get_framerate() == tag.fps) &&
                     (tag.stream_index == -1 || vp->get_stream_index() == tag.stream_index))
```
But better formatted :)

Reply:
Done